vignettes/ebola.Rmd

```diff
@@ -0,0 +1,620 @@
+)
+```
+
+```{r}
+draws_sex_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex-district:
+
+```{r}
+newdata_sex_district_pmf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex, district) |>
+  mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_district_pmf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pmf,
+  ndraws = 1000
+)
+```
+
+```{r}
+draws_sex_district_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+### Continuous probability density function
+
+The posterior predictive distribution under no truncation and no censoring.
+That is to produce continuous delay times.
+I don't like that `geom_density()` is doing kernel smoothing but plotting as a histogram is suggestive of there being discrete structure.
+
+No stratification:
+
+```{r}
+newdata_pdf <- obs_prep |>
+  as.data.frame() |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_pdf <- predicted_draws(fit, newdata = newdata_pdf, ndraws = 1000)
+```
+
+```{r}
+ggplot(draws_pdf, aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pdf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex) |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_pdf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pdf,
+  ndraws = 1000
+)
+```
+
+```{r}
+draws_sex_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_district_pdf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex, district) |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_district_pdf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pdf,
+  ndraws = 1000
+)
+
+```{r}
+draws_sex_district_pdf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+## Model comparison
+
+* Use `loo` and PSIS
```

Comment:
This is already a super long vignette. There's a lot in here, but I think few will make it this far.

I'd recommend keeping this really short. Focus on the population level only.

Reply:
Has been cut into https://github.com/epinowcast/epidist/issues/285